dynet/simd-functors.h

```diff
@@ -71,6 +71,30 @@ struct functor_traits<dynet::const_minus_op<Scalar> > {
 } }
 
 namespace dynet {
+template<typename Scalar> struct scalar_negative_op {
```

Comment:
If the above change to `nodes.cc` works, then this addition may not be necessary.